relay/channel/replicate/adaptor_polling_test.go

```
@@ -0,0 +1,197 @@
+		pollCount.Add(1)
+		assert.Equal(t, "/v1/predictions/prediction-1", r.URL.Path)
+		assert.Equal(t, "Bearer replicate-token", r.Header.Get("Authorization"))
+		w.Header().Set("Content-Type", "application/json")
+		_, err := io.WriteString(w, `{"id":"prediction-1","status":"succeeded","output":["https://example.com/one.png","https://example.com/two.png"],"error":null}`)
+		assert.NoError(t, err)
+	}))
+	defer server.Close()
+
+	recorder := httptest.NewRecorder()
+	c, _ := gin.CreateTestContext(recorder)
+	c.Request = httptest.NewRequest(http.MethodPost, "/v1/images/generations", nil)
```

golangci-lint (2.12.2)

[error] net/http/httptest.NewRequest must not be called. use net/http/httptest.NewRequestWithContext

(noctx)